src/scss/_utils.scss

```
@@ -51,18 +51,22 @@
   margin-top: 0.25rem;
 }
 
-// Limit the width of tooltip to 20rem.
-// 25rem was chosen as the breakpoint, because 80vw equals 20rem at 25rem.
 .p-tooltip--constrain-width .p-tooltip__message {
-  @media (width <= 25rem) {
-    width: 80vw;
-  }
+  max-width: 20rem;
+  white-space: pre-wrap;
 
-  @media (width > 25rem) {
-    width: 20rem;
+  @media (max-width: $breakpoint-x-small) {
+    max-width: 80vw;
   }
+}
 
-  white-space: pre-wrap;
+// Specifies horizontal width of tooltips with 0 width on small screens.
+// Combine with utils/breakLines for tooltips with 0 width on bigger width screens.
+.p-tooltip--fixed-x-small-width .p-tooltip__message {
+  @media (max-width: $breakpoint-x-small) {
+    white-space: pre-wrap;
```

Because this has a fixed width we can just use `wrap` here to avoid having to conditionally breaking the lines in the component.

```suggestion
    white-space: wrap;
```